install/install.md

```diff
@@ -11,8 +11,7 @@ description: |
 {: .fs-8 }
 
 RAPIDS has several methods for installation, depending on the preferred environment and version. <br/>
-*New Users* should be mindful of the system and environment prerequisites. 
-{: .fs-6 .fw-300 }
```

Comment:
The phrase "be mindful of" sounds a little bit intense to me. Like "be mindful of _&lt;something dangerous&gt;_."

Also the font size for this felt aggressively large. It's text, not a heading.